src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -89,12 +100,14 @@ List<ContingencyListAttributes> getContingencyLists() {
     }
 
     List<ContingencyListAttributes> getContingencyLists(List<UUID> ids) {
-        return Stream.concat(
+        return Stream.of(
```

Comment:
Why ?

Reply:
Stream.concat() takes only 2 arguments. I need to concat 3 streams here